src/code/utilities/genetics-utils.js

```diff
@@ -471,4 +471,59 @@ export default class GeneticsUtils {
     });
     return {geneStart, chromosomeHeight: species.chromosomesLength[chromosomeName]};
   }
+
+  /**
+   * Given a gene string, returns an object representing the phenotypes expressed by that gene string. For example,
+   * 'a:w,b:w' is converted to {'wings': 'No wings'}.
+   */
+  static convertGeneStringToPhenotype(genes) {
```

Comment:
I'm not sure, I agree using traits instead would be simpler. I didn't want to change all the grading code, but that's probably the correct solution. 

Rather than saying something like:

`{ "label": "Drakes with forelimbs and hindlimbs", "alleles": ["Fl-, Hl-","Fl-, -Hl","-Fl, Hl-","-Fl, -Hl"] }`

It would probably be simpler to write:

`{ "label": "Drakes with forelimbs and hindlimbs", "phenotypes": [{"forelimbs": "Forelimbs", "hindlimbs": "Hindlimbs"] }`